network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/hero.html

```diff
@@ -9,9 +9,9 @@
     </div>
 
     <div class="tw-relative tw-space-y-5 medium:tw-space-y-6 tw-flex tw-flex-col tw-justify-center tw-items-start">
-      <h1 class="tw-mb-0 tw-leading-none tw-font-bold tw-text-4xl medium:tw-text-[3.25rem]">{% trans "Does this button work?" %}</h1>
-      <p class="tw-body-large large:tw-text-2xl">
-        <span class="tw-font-bold">{% trans "Investigating YouTube’s ineffective user controls" %}</span><br/>
+      <h1 class="mb-0 tw-leading-none tw-font-bold tw-text-4xl medium:tw-text-[3.25rem]">{% trans "Does this button work?" %}</h1>
+      <p class="mb-1 tw-text-3xl tw-font-bold mt-0">{% trans "Investigating YouTube’s ineffective user controls" %}</p>
+      <p class="tw-body-large large:tw-text-2xl mt-0">
```

Comment:
**Issue** We should stay away from using the Bootstrap utilities and rather use the Tailwind ones (`tw-`).
```suggestion
      <h1 class="tw-mb-0 tw-leading-none tw-font-bold tw-text-4xl medium:tw-text-[3.25rem]">{% trans "Does this button work?" %}</h1>
      <p class="tw-mb-1 tw-text-3xl tw-font-bold tw-mt-0">{% trans "Investigating YouTube’s ineffective user controls" %}</p>
      <p class="tw-body-large large:tw-text-2xl tw-mt-0">
```